packages/text-serializer/lib/adapter.js

```diff
@@ -0,0 +1,18 @@
+const serializeText = require('./serializeText');
+
+const name = 'text';
+const mediaTypes = [
+  'text/plain',
```

Comment:
I think we should match on `text/*`, i.e, any string element containing `<html>...</html>` as content with the content-type `text/html` should work, etc.

This PR probably isn't the place to solve this, so I'd probably just put something in our backlog to solve in the future. I think this will require changing how the core works.